tls/rotation.go

```diff
@@ -0,0 +1,167 @@
+package tls
+
+import (
+	"time"
+)
+
+var (
+	kubeControlPlaneSignerConfig = SignerConfig{
+		Namespace: "openshift-kube-apiserver-operator", Name: "kube-control-plane-signer",
+		Validity: 60 * 24 * time.Hour, // this comes from the installer TODO: what does this mean?
```

Comment:
i wonder if we can use something like https://github.com/xhit/go-str2duration and write down "60d" here :-)